src/app/components/Item/ItemFilters.jsx

```diff
@@ -191,7 +190,7 @@ const ItemFilters = (
   };
   // If there are filters, display the number of items that match the filters.
   // Otherwise, display the total number of items.
-  const resultsItemsNumber = selectedFilterDisplayStr ? numItemsCurrent : numItemsTotal;
```

Comment:
Shouldn't this be `numItemsMatched : numItemsTotal`? I thought that `numItemsMatched` would return 0 _unless_ if filters were applied. So on the first load without filters, wouldn't this be 0 if it was set to `numItemsMatched`?

Reply:
numitemsmatched is always returned from the bib findbyuri endpoint (most if not all calls on the bib page are this endpoint)